ironfish/src/wallet/wallet.test.slow.ts

```diff
@@ -1251,6 +1251,7 @@ describe('Wallet', () => {
       expect(addResult2.isAdded).toBeTruthy()
 
       await node.wallet.updateHead()
+      await node.wallet.scanTransactions()
```

Comment:
nit - I think we should be able to move this call to right after all of the account imports

one thing we're running into here is that importing an account through the sdk doesn't automatically trigger a scan (but importing through rpc does)